deploy/charts/harvester/templates/cattle-system.yaml

```diff
@@ -0,0 +1,24 @@
+# system-upgrade-controller watches secret and node change.
+# If there is a new node, system-upgrade-controller creates a job on the node to change CA certificates.
+# If additional-ca is changed, system-upgrade-controller creates a job on each node to change CA certificates.
+apiVersion: upgrade.cattle.io/v1
+kind: Plan
+metadata:
+  name: sync-additional-ca
+  namespace: cattle-system
+spec:
+  concurrency: 1
+  nodeSelector:
+    matchLabels:
+      harvesterhci.io/managed: "true"
+  serviceAccountName: system-upgrade-controller
+  secrets:
+    - name: harvester-additional-ca
+      path: /ca
+  version: v1.0.2
+  upgrade:
+    image: {{ .Values.syncAdditionalCA.image.repository }}:{{ .Values.syncAdditionalCA.image.tag }}
+    command: ["/bin/sh", "-c"]
+    args:
+      - cp /ca/additional-ca.pem /host/etc/pki/trust/anchors;
+      - chroot /host update-ca-certificates
```

Comment:
The initial additional-ca secrete is empty, will it also trigger here, should it be excluded ?

meanwhile, is it possible to delete additional-ca ?

Reply:
Yes, it also triggers, but it's okay. An empty additional-ca will remove the last added additional-ca, so we can pass the test case 3 `Remove additional-ca`.